src/Apps/CollectorProfile/Components/CollectorProfileHeader/Components/CollectorProfileHeaderInfo.tsx

```diff
@@ -60,7 +56,7 @@ interface InfoFieldProps {
 }
 
 const InfoField: FC<InfoFieldProps> = ({ type, value }) => {
-  let Icon: FC<IconProps> = Fragment
+  let Icon: FC<ComponentProps<typeof InstitutionIcon>> = Fragment
```

Comment:
I keep seeing this pattern: this pretty annoying to deal with. I'd avoid this completely — a config object would be fine here.